backend/src/ml_space_lambda/data_access_objects/group.py

```diff
@@ -35,13 +35,15 @@ def __init__(
         created_by: str,
         created_at: Optional[float] = None,
         last_updated_at: Optional[float] = None,
+        num_members: Optional[int] = None,
```

Comment:
Why are we adding `num_members` do the DAO if it is only computed dynamically in the group listing page?

Reply:
So that `numMembers` can be populated in the dictionary/JSON we send to the frontend

Reply:
I like your suggestion, just committed it

Reply:
I'll make another commit in a minute here removing the numGroup property from the DAO